### ISSUE TITLE
requesting raw message results in json decoding error

### ISSUE THREAD

**Issue by tpetry** [contextio/ContextIO-node#41]
It seems that the library is trying to json decode a raw email message:

```js
const ctxioClient = ContextIO({key: '...', secret: '...', version: 'lite'})
ctxioClient
  .users('...')
  .email_accounts('...')
  .folders('...')
  .messages('...')
  .raw()
  .get()
  .then(console.log)
```

    Unhandled rejection SyntaxError: Unexpected token D in JSON at position 0
        at JSON.parse (<anonymous>)
        at sendRequest.then.body (/Users/tpetry/Documents/contextio/test005/node_modules/contextio/lib/call-service.js:57:21)
        at tryCatcher (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/util.js:16:23)
        at Promise._settlePromiseFromHandler (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/promise.js:512:31)
        at Promise._settlePromise (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/promise.js:569:18)
        at Promise._settlePromise0 (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/promise.js:614:10)
        at Promise._settlePromises (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/promise.js:693:18)
        at Async._drainQueue (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/async.js:133:16)
        at Async._drainQueues (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/async.js:143:10)
        at Immediate.Async.drainQueues (/Users/tpetry/Documents/contextio/test005/node_modules/bluebird/js/release/async.js:17:14)
        at runCallback (timers.js:781:20)
        at tryOnImmediate (timers.js:743:5)
        at processImmediate [as _immediateCallback] (timers.js:714:5)

**Comment by tpetry** [comment]
The change fixes the bug. Could you tag a new release and publish it to npm?

**Comment by AaronLaBrie** [comment]
@tpetry Thanks for the bug report! Pushed out a fix for this `v1.2.1`, should be out on npm now.

**Comment by tpetry** [comment]
It's available at npm. Thank you.
